android/src/main/java/com/iyaffle/launchreview/LaunchReviewPlugin.java

```
@@ -1,135 +1,54 @@
-        }  else {
-            result.notImplemented();
-        }
-    }
-
-    @Override
-    public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {
-        register(this, binding.getBinaryMessenger(), null);
-    }
-
-    @Override
-    public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
-
-    }
-
-    @Override
-    public void onAttachedToActivity(@NonNull ActivityPluginBinding binding) {
-        activity = binding.getActivity();
-    }
-
-    @Override
-    public void onDetachedFromActivityForConfigChanges() {
-
-    }
-
-    @Override
-    public void onReattachedToActivityForConfigChanges(@NonNull ActivityPluginBinding binding) {
-
-    }
-
-    @Override
-    public void onDetachedFromActivity() {
-
-    }
+public class LaunchReviewPlugin implements MethodCallHandler {
+
+  private final Registrar mRegistrar;
+
+  private LaunchReviewPlugin(Registrar registrar) {
+    this.mRegistrar = registrar;
+  }
+
+  /**
+   * Plugin registration.
+   */
+  public static void registerWith(Registrar registrar) {
+    final MethodChannel channel = new MethodChannel(registrar.messenger(), "launch_review");
+    LaunchReviewPlugin instance = new LaunchReviewPlugin(registrar);
+    channel.setMethodCallHandler(instance);
+  }
+
+  @Override
+  public void onMethodCall(MethodCall call, Result result) {
+    if (call.method.equals("launch")) {
+
+      String appId = (String) call.argument("android_id");
+      String appPackageName;
+
+      if (appId != null) {
+        appPackageName = appId;
+      } else {
+        appPackageName = mRegistrar.activity().getPackageName();
+      }
+
+      Intent marketIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=" + appPackageName));
+      marketIntent.addFlags(
+          Intent.FLAG_ACTIVITY_NO_HISTORY | Intent.FLAG_ACTIVITY_NEW_DOCUMENT | Intent.FLAG_ACTIVITY_MULTIPLE_TASK);
+      mRegistrar.activity().startActivity(marketIntent);
```

This removes critical error handling and fallback logic:

**Master branch has:**
- Detection of Google Play Store installation
- Fallback to web browser if Play Store not available  
- Try-catch for ActivityNotFoundException
- Toast notification to user

**This PR removes all of that**, replacing it with a simple intent that will fail silently on devices without Play Store (common in China, custom ROMs, etc.).

Recommend keeping the master branch implementation for better reliability.